docs/edgeql/insert.rst

```diff
@@ -255,6 +255,11 @@ conflicts and provide a fallback expression.
   apply additional filters on ``Movie``; in the context of the ``else`` clause,
   ``Movie`` is bound to the conflicting object.
 
+.. note::
+
+    Although 2.x does *not* support ``unless conflict`` on :ref:`multi
```

Comment:
I think this might be more readable if inverted - "using `unless conflict` on multi properties is only supported in 3.0 and later"